upup/pkg/fi/cloudup/hetznertasks/server.go

```diff
@@ -33,7 +33,7 @@ import (
 type ServerGroup struct {
 	Name      *string
 	Lifecycle fi.Lifecycle
-	SSHKey    *SSHKey
+	SSHKeys   []*SSHKey
```

Comment:
Do we set this anywhere?  I didn't see if in the diff, which is surprising (?)

Reply:
We don't set this anywhere, except for model. They are used only when creating a new server and otherwise ignored.
Hetzner Cloud API doesn't return the keys for a running server.